weave-js/src/components/PagePanel.tsx

```diff
@@ -185,12 +186,28 @@ const usePoorMansLocation = () => {
   return window.location;
 };
 
+const useInitializeDataDog = () => {
+  const weaveViewer = useWeaveViewer();
+
+  useEffect(() => {
+    if (weaveViewer.loading) {
+      return;
+    }
+    const userInfo = {username: 'anonymous'};
```

Comment:
It looks like [someone has registered](https://wandb.ai/anonymous) with anonymous as their username...

Reply:
Done